src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -102,6 +107,14 @@ public void save(AddressBook addressBook) throws StorageOperationException {
             throw new StorageOperationException("Error converting address book into storage format");
         }
     }
+    
+    // should I put a comment for this method? Since this method is rather self explanatory.
+	private void checkIfFileExists() {
+		File file = path.toFile();
+    	if (file == null || !file.exists()) {
+    		new TextUi().showToUser(MESSAGE_ERROR_FILE_DOES_NOT_EXIST);
```

Comment:
- indentation is off in this method.
- this is not really the right place to show the message, the message should be propagated to some place where the text UI object is already handling user interactions. 
- as you will see in further lectures, you are unnecessarily creating a dependency on `TextUI` in `Storage`.